app/modules/flappybird/endpoints_flappybird.py

```diff
@@ -104,12 +104,36 @@ async def create_flappybird_score(
         user_id=user.id,
         value=flappybird_score.value,
         creation_time=creation_time,
-        # We add all informations contained in the schema
     )
-    try:
-        return await cruds_flappybird.create_flappybird_score(
+    db_flappybird_best_score = models_flappybird.FlappyBirdBestScore(
+        id=score_id,
+        user_id=user.id,
+        value=flappybird_score.value,
+        creation_time=creation_time,
+    )
+    personal_best = await cruds_flappybird.get_flappybird_personal_best_by_user_id(
+        user_id=user.id,
+        db=db,
+    )
+    if not personal_best:
+        try:
```

Comment:
I'm not sure try/except is really useful as they don't do anything apart from raising a new exception which won't be logged. I think you could remove them